signals.mml

```diff
@@ -408,6 +419,7 @@ Layer:
           ORDER BY
             -- distant signals are less important, signals for slower speeds are more important
             (CASE
+              WHEN railway_has_key(tags, 'railway:signal:departure') THEN 15000
```

Comment:
Assigning 15000 to departure signals will make them rendered with higher priority than main signals. Is this intended?

Reply:
If I read the query correctly, the high numbers are low priority (rendered last, without overlap). The values are sorted ascending, so 15000 comes last.

2 screenshots for validation:

http://127.0.0.1:6789/openrailwaymap/#16/52.2577/6.1607
![image](https://github.com/OpenRailwayMap/OpenRailwayMap-CartoCSS/assets/1073881/a2921b3d-3fc7-46d0-8e7a-3aae0efc4a9e)

and zoomed out one level

http://127.0.0.1:6789/openrailwaymap/#15/52.2578/6.1612
![image](https://github.com/OpenRailwayMap/OpenRailwayMap-CartoCSS/assets/1073881/2c519c3f-352d-45e7-aa39-2768cb9ec17d)

Signal with ref `116` is rendered instead of the departure light.